### PR TITLE
Update icon grids to put more high profile apps on the desktop

### PR DESCRIPTION
**Pull request by nedrichards**

### CODE REVIEW

File: data/settings/icon-grid-C.json.in [hunk 1]
```diff
@@ -3,25 +3,28 @@
...
-    "eos-folder-work.directory",
-    "eos-folder-curiosity.directory",
-    "eos-folder-social.directory"
+    "eos-folder-media.directory",
+    "eos-folder-social.directory",
+    "yelp.desktop"
   ],
   "eos-folder-media.directory" : [
+    "com.endlessm.photos.desktop",
+    "com.endlessm.videonet.desktop",
```
Comment (rshuler) [inline data/settings/icon-grid-C.json.in]: Videonet no longer is supported.

File: data/settings/icon-grid-C.json.in [hunk 2]
```diff
@@ -36,29 +39,8 @@
     "net.sourceforge.Extremetuxracer.desktop",
     "net.sourceforge.Warmux.desktop"
   ],
-  "eos-folder-work.directory" : [
-    "libreoffice-calc.desktop",
-    "libreoffice-impress.desktop",
-    "gnome-calculator.desktop",
```
Comment (rshuler) [inline data/settings/icon-grid-C.json.in]: Confirming that we don't want these apps in any default folder anymore: calculator, finance, resume, translation?

File: data/settings/icon-grid-C.json.in [hunk 3]
```diff
@@ -36,29 +39,8 @@
...
-    "com.endlessm.finance.desktop",
-    "com.endlessm.resume.desktop",
-    "com.endlessm.translation.desktop"
-  ],
-  "eos-folder-curiosity.directory" : [
```
Comment (rshuler) [inline data/settings/icon-grid-C.json.in]: Really!?  No knowledge apps on the desktop / in a folder by default?

File: data/settings/icon-grid-C.json.in [hunk 4]
```diff
@@ -36,29 +39,8 @@
...
-    "com.endlessm.cooking.en.desktop",
-    "com.endlessm.celebrities.en.desktop",
-    "com.endlessm.myths.en.desktop",
-    "com.endlessm.math.en.desktop",
-    "com.endlessm.howto.en.desktop",
-    "com.endlessm.travel.en.desktop",
-    "com.endlessm.health.en.desktop",
-    "kde4-org.kde.Marble.desktop",
-    "eos-link-duolingo.desktop"
```
Comment (rshuler) [inline data/settings/icon-grid-C.json.in]: Confirming that we don't want duolingo in any folder by default?

File: data/settings/icon-grid-C.json.in (hunk 1 above)
Comment (nedrichards) [inline data/settings/icon-grid-C.json.in]: OK, will remove.

**Comment by nedrichards** [comment]
The icon grids aren't identical as they all have different language encyclopedias I initially made that mistake myself! 

We don't have all the apps that were designed into the 3 line grid yet, so we'll need to do updates as those apps are produced to fill it out. 

Good point on the existing knowledge apps - some of these we do want to keep, this is what happens when I expand the scope of a task from 'update indonesia, thailand and vietnam' to 'everything else seems easy'. I'll revert some of the other files and consider them for a separate PR.

It was intentional not to touch zh_CN as that's not a market we're putting any active work into right now. I'm aware that there are local constraints and I don't want to make things any worse without doing my research.

File: data/settings/icon-grid-C.json.in (hunk 2 above)
Comment (nedrichards) [inline data/settings/icon-grid-C.json.in]: I'm adding translation back - but yes we're dropping resume and finance - their usage is very low. There's not currently a sane place to put the calculator and I don't want it on top level as again, usage is low.

File: data/settings/icon-grid-C.json.in (hunk 1 above)
Comment (nedrichards) [inline data/settings/icon-grid-C.json.in]: Removing

File: data/settings/icon-grid-C.json.in (hunk 4 above)
Comment (nedrichards) [inline data/settings/icon-grid-C.json.in]: Have re-added this folder for now until we have new apps

File: data/settings/icon-grid-C.json.in [hunk 5]
```diff
@@ -3,24 +3,41 @@
...
+    "eos-link-whatsapp.desktop",
     "eos-link-youtube.desktop",
     "eos-folder-media.directory",
-    "eos-folder-games.directory",
-    "eos-folder-work.directory",
     "eos-folder-curiosity.directory",
-    "eos-folder-social.directory"
+    "eos-folder-games.directory",
+    "eos-folder-social.directory",
+    "net.minetest.Minetest.desktop",
```
Comment (nedrichards) [inline data/settings/icon-grid-C.json.in]: This is intentional and indeed expected, whenever the user adds an app to the desktop it'll be in the 'last' slot so this scenario has to look good, assuming we're successful in our goals.

File: data/settings/icon-grid-C.json.in [hunk 6]
```diff
@@ -3,24 +3,41 @@
...
   ],
+  "eos-folder-curiosity.directory" : [
+    "com.endlessm.celebrities.en.desktop",
+    "com.endlessm.myths.en.desktop",
+    "com.endlessm.math.en.desktop",
+    "com.endlessm.howto.en.desktop",
+    "com.endlessm.travel.en.desktop",
+    "com.endlessm.dinosaurs.en.desktop",
+    "com.endlessm.health.en.desktop",
+    "kde4-org.kde.Marble.desktop",
```
Comment (nedrichards) [inline data/settings/icon-grid-C.json.in]: I'll remove for now - this is actually from me having the KDE repos installed on my system - but you're right, we really should enable maps.

File: data/settings/icon-grid-C.json.in (hunk 3 above)
Comment (nedrichards) [inline data/settings/icon-grid-C.json.in]: Not intentional and not reflected in the latest patch

File: data/settings/icon-grid-es_GT.json.in [hunk 7]
```diff
@@ -3,24 +3,28 @@
...
+    "net.minetest.Minetest.desktop",
+    "com.endlessm.cooking.es.desktop",
+    "rhythmbox.desktop"
   ],
   "eos-folder-media.directory" : [
     "shotwell.desktop",
+    "com.endlessm.photos.desktop",
     "org.gnome.Totem.desktop",
-    "net.sourceforge.Audacity.desktop",
     "eos-link-clarovideo.desktop",
```
Comment (nedrichards) [inline data/settings/icon-grid-es_GT.json.in]: Done

File: data/settings/icon-grid-es_GT.json.in [hunk 8]
```diff
@@ -3,24 +3,28 @@
...
     "eos-link-claromusica.desktop",
     "org.tuxpaint.Tuxpaint.desktop",
```
Comment (nedrichards) [inline data/settings/icon-grid-es_GT.json.in]: None, added

File: data/settings/icon-grid-es_GT.json.in [hunk 9]
```diff
@@ -38,14 +42,6 @@
...
   ],
-  "eos-folder-work.directory" : [
-    "libreoffice-calc.desktop",
-    "libreoffice-impress.desktop",
-    "gnome-calculator.desktop",
-    "com.endlessm.finance.desktop",
-    "com.endlessm.resume.desktop",
-    "com.endlessm.translation.desktop"
-  ],
   "eos-folder-curiosity.directory" : [
```
Comment (nedrichards) [inline data/settings/icon-grid-es_GT.json.in]: Done

File: data/settings/icon-grid-es_GT.json.in [hunk 10]
```diff
@@ -3,24 +3,28 @@
...
-    "eos-folder-social.directory"
+    "eos-folder-games.directory",
+    "eos-folder-social.directory",
+    "net.minetest.Minetest.desktop",
+    "com.endlessm.cooking.es.desktop",
+    "rhythmbox.desktop"
   ],
   "eos-folder-media.directory" : [
     "shotwell.desktop",
+    "com.endlessm.photos.desktop",
```
Comment (nedrichards) [inline data/settings/icon-grid-es_GT.json.in]: Done

File: data/settings/icon-grid-es.json.in [hunk 11]
```diff
@@ -53,12 +50,11 @@
     "kde4-org.kde.Marble.desktop",
     "eos-link-duolingo.desktop",
     "com.endlessm.maternity.es.desktop",
-    "com.endlessm.world_literature.es.desktop"
+    "com.endlessm.dinosaurs.es,
```
Comment (nedrichards) [inline data/settings/icon-grid-es.json.in]: Done

File: data/settings/icon-grid-es.json.in [hunk 12]
```diff
@@ -53,12 +50,11 @@
     "kde4-org.kde.Marble.desktop",
     "eos-link-duolingo.desktop",
```
Comment (nedrichards) [inline data/settings/icon-grid-es.json.in]: Done

File: data/settings/icon-grid-es.json.in [hunk 13]
```diff
@@ -36,14 +41,6 @@
...
   ],
-  "eos-folder-work.directory" : [
-    "libreoffice-calc.desktop",
-    "libreoffice-impress.desktop",
-    "gnome-calculator.desktop",
-    "com.endlessm.finance.desktop",
-    "com.endlessm.resume.desktop",
-    "com.endlessm.translation.desktop"
-  ],
   "eos-folder-curiosity.directory" : [
```
Comment (nedrichards) [inline data/settings/icon-grid-es.json.in]: Done

File: data/settings/icon-grid-es.json.in [hunk 14]
```diff
@@ -3,22 +3,27 @@
...
+    "rhythmbox.desktop"
   ],
   "eos-folder-media.directory" : [
+    "com.endlessm.photos.desktop",
     "shotwell.desktop",
     "org.gnome.Totem.desktop",
-    "net.sourceforge.Audacity.desktop",
     "org.tuxpaint.Tuxpaint.desktop",
+    "net.sourceforge.Audacity.desktop",
+    "com.endlessm.world_literature.es",
```
Comment (nedrichards) [inline data/settings/icon-grid-es.json.in]: Done

File: data/settings/icon-grid-id.json.in [hunk 15]
```diff
@@ -3,22 +3,24 @@
...
+    "libreoffice-impress.desktop",
     "com.endlessm.encyclopedia.id.desktop",
-    "net.minetest.Minetest.desktop",
+    "eos-link-facebook.desktop",
+    "eos-link-whatsapp.desktop",
     "eos-link-youtube.desktop",
     "eos-folder-media.directory",
     "eos-folder-games.directory",
-    "eos-folder-work.directory",
-    "eos-folder-curiosity.directory",
```
Comment (nedrichards) [inline data/settings/icon-grid-id.json.in]: I was planning to add the apps when they were available, the plan for indonesia is to add as many of them onto the desktop as will fit before overflowing into a folder and I didn't want to add a folder just for translation and duolingo given that maps doesn't actually exist. Not married to this position but I'd expect an update here before we actually ship 3.2.0 when the apps are available.

File: data/settings/icon-grid-pt.json.in [hunk 16]
```diff
@@ -36,28 +41,21 @@
...
   ],
-  "eos-folder-work.directory" : [
-    "libreoffice-calc.desktop",
-    "libreoffice-impress.desktop",
-    "gnome-calculator.desktop",
-    "com.endlessm.finance.desktop",
-    "com.endlessm.resume.desktop",
-    "com.endlessm.translation.desktop"
-  ],
   "eos-folder-curiosity.directory" : [
```
Comment (nedrichards) [inline data/settings/icon-grid-pt.json.in]: Done

File: data/settings/icon-grid-th.json.in [hunk 17]
```diff
@@ -3,22 +3,24 @@
...
+    "libreoffice-impress.desktop",
     "com.endlessm.encyclopedia.th.desktop",
-    "net.minetest.Minetest.desktop",
+    "eos-link-facebook.desktop",
+    "eos-link-whatsapp.desktop",
     "eos-link-youtube.desktop",
     "eos-folder-media.directory",
     "eos-folder-games.directory",
-    "eos-folder-work.directory",
-    "eos-folder-curiosity.directory",
```
Comment (nedrichards) [inline data/settings/icon-grid-th.json.in]: Same as above - not yet

File: data/settings/icon-grid-vi.json.in [hunk 18]
```diff
@@ -3,22 +3,25 @@
...
+    "libreoffice-impress.desktop",
     "com.endlessm.encyclopedia.vi.desktop",
-    "net.minetest.Minetest.desktop",
+    "eos-link-facebook.desktop",
+    "eos-link-whatsapp.desktop",
     "eos-link-youtube.desktop",
     "eos-folder-media.directory",
     "eos-folder-games.directory",
-    "eos-folder-work.directory",
-    "eos-folder-curiosity.directory",
```
Comment (nedrichards) [inline data/settings/icon-grid-vi.json.in]: Same as above - not yet

File: data/settings/icon-grid-C.json.in (hunk 5 above)
Comment (rshuler) [inline data/settings/icon-grid-C.json.in]: Okay, but keep in mind that we typically use the default layout for screenshots on the website and marketing materials, so in some sense how it looks out of the box should have a little more emphasis.  But let's go ahead and see how it looks.

**Pull request merged**